src/main/scala/com/microsoft/partnercatalyst/fortis/spark/analyzer/timeseries/Area.scala

```diff
@@ -0,0 +1,44 @@
+package com.microsoft.partnercatalyst.fortis.spark.analyzer.timeseries
+
+trait AreaFinder {
+  def findArea(tilez: Int, tilex: Int, tiley: Int): Option[Area]
```

Comment:
@erikschlegel this trait may already exist somewhere in your code. It represents a black box which takes tiles coordinates and gives you a general are for which it belongs. A simple implementation would be to have the coordinates `mod` to a tile size and a more complex one could use something like clustering to determine more appropriate area given existing activity.